shards/shards_test.go

```diff
@@ -244,3 +244,78 @@ func searcherForTest(t *testing.T, b *zoekt.IndexBuilder) zoekt.Searcher {
 
 	return searcher
 }
+
+func reposForTest(n int) (result []*zoekt.Repository) {
+	for i := 0; i < n; i++ {
+		result = append(result, &zoekt.Repository{
+			Name: fmt.Sprintf("test-repository-%d\n", i),
```

Comment:
intentional to have a new line in the repo name?

Reply:
Fixed, thanks!